### PR TITLE
Passed props to wrapper div for added styling flexibility

### PR DESCRIPTION
**Pull request by abohannon**
Slight refactor to pass wrapper styling props to the button container div.

Updated examples/src/index.js to demo button

I'm not familiar with Flow and my `wrapperStyle?: object` is being rejected in the tests. Happy to update, I'd just need some direction on the appropriate type declaration for this prop.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -146,6 +146,3 @@ typings/
 
 # End of https://www.gitignore.io/api/node,jetbrains+all
 #
-
-## CUSTOM
-dist/
```
Comment (tysonbattistella) [inline .gitignore]: Any reason this was removed? We probably don't need to have version control on our build output.

File: src/index.js [hunk 2]
```diff
@@ -15,8 +15,10 @@ type Props = {
   onChargeFailure?: (MessageData) => void,
   onPaymentDetected?: (MessageData) => void,
   onModalClosed?: () => void,
-  disableCaching: true
+  disableCaching: true,
+  wrapperStyle?: object,
```
Comment (tysonbattistella) [inline src/index.js]: `wrapperStyle?: { [string]: number | string }` should do the trick

**Comment by tysonbattistella** [comment]
Thanks for the PR! I've left a few comments to clean it up and fix the tests, but looks good overall 👍

**Comment by abohannon** [comment]
Sorry about that. Forgot that maintaining my own fork would be reflected in the PR. Needed dist/ for use in my own project. Updated the PR with your requested changes. All tests passing. Thanks for the review!

**Comment by tysonbattistella** [comment]
@abohannon Thanks! Can you remove the `dist` directory from your PR? I see the `.gitignore` was updated, but the directory is still checked into Git.

**Comment by abohannon** [comment]
@tysonbattistella my mistake. Should be good now!

**Pull request merged**